src/Illuminate/Auth/Access/Gate.php

```diff
@@ -123,19 +128,27 @@ public function define($ability, $callback)
      * @param  string  $name
      * @param  string  $class
      * @param  array   $abilities
+     * @param  array   $options
      * @return $this
      */
-    public function resource($name, $class, array $abilities = null)
+    public function resource($name, $class, array $abilities = [], array $options = [])
     {
         $abilities = $abilities ?: [
+            'index'   => 'index',
```

Comment:
Resource makes more since with the `index` ability and wasn't a good idea to add it before because most of the time you need to enable it for guests .. right ?